decidim-meetings/app/views/decidim/meetings/meetings/_meetings.html.erb

```diff
@@ -1,5 +1,17 @@
+  <% if @meetings_warning %>
```

Comment:
@josepjaume but following this logic we could have a different message based on the filters you have selected.